hydromt_wflow/utils.py

```diff
@@ -494,36 +630,51 @@ def _update_output_netcdf_grid(wflow_var, var_name):
     }
     for section, options in input_section.items():
         for key in options:
-            value = get_config(key, config=config, fallback=None)
+            value = get_config(config, key, fallback=None)
             if value is not None:
                 if section == "general":
+                    # TODO this is bad, but conversion functionality will change
+                    # in the future, so it will do for now
```

Comment:
I think that's okay. It's the only argument in the genral section that was renamed so no need to be too fancy :)